atd-etl/cris_import/cris_import.py

```diff
@@ -225,45 +295,91 @@ def specify_extract_location():
     return zip_tmpdir
 
 
-def download_archives():
-    """
-    Connect to the SFTP endpoint which receives archives from CRIS and
-    download them into a temporary directory.
+def download_s3_archive():
+    session = boto3.Session(
+        aws_access_key_id=AWS_ACCESS_KEY_ID,
+        aws_secret_access_key=AWS_SECRET_ACCESS_KEY,
+    )
 
-    Returns path of temporary directory as a string
-    """
+    s3 = session.client("s3")
+    bucket = S3_EXTRACT_BUCKET
 
     with SshKeyTempDir() as key_directory:
         write_key_to_file(
-            key_directory + "/id_ed25519", SFTP_ENDPOINT_SSH_PRIVATE_KEY + "\n"
+            key_directory + "/id_ed25519", DB_BASTION_HOST_SSH_PRIVATE_KEY + "\n"
```

Comment:
This is a real confusing part of this diff. It's managed to intermingle two parts of the program that did wildly different things:
* Connecting to the sftp host to rsync files down
* Connect to the bastion to tunnel into the DB server

It's bonkers how it diff's like this.